homework/content.js

```diff
@@ -1,21 +1,24 @@
 'use strict';
 
+
 (function() {
-  var _wrapper = document.querySelector('#note-content-wrapper');
+	function contentObject() {
+			this._wrapper = document.querySelector('#note-content-wrapper');
+	}
 
-  function start() {
+  contentObject.prototype.start = function() {
```

Comment:
It's a convention to give such constructor as `ContentObject` rather than `contentObject` -- it's confused with the instance.